pkg/system/reconciler.go

```diff
@@ -295,6 +297,10 @@ func NewReconciler(
 	// Setting default AWS STS cluster as false
 	r.IsAWSSTSCluster = false
 
+	// Set bucket logging volume mount name and path
+	r.BucketLoggingVolume = r.Request.Name + "bucket-logging-volume"
```

Comment:
Add a `-` in the middle
```suggestion
	r.BucketLoggingVolume = r.Request.Name + "-bucket-logging-volume"
```

Reply:
Sure, Thanks.